src/main/java/seedu/address/model/good/Good.java

```diff
@@ -42,6 +52,19 @@ public static boolean isValidGood(Good test) {
                 && GoodQuantity.isValidGoodQuantity(test.getGoodQuantity().toString());
     }
 
+    public GoodQuantity getThreshold() {
+        return threshold;
+    }
+
+    /**
+     * Checks whether the quantity of the good is lower or equal to the threshold.
+     *
+     * @return true if lower than threshold.
+     */
+    public boolean quantityLowerThanThreshold() {
```

Comment:
The name sounds slightly misleading, any thoughts?

Reply:
lower or equals to. i see.